examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCCastingPlayer.h

```diff
@@ -48,32 +48,34 @@
 - (void)verifyOrEstablishConnectionWithCompletionBlock:(void (^_Nonnull)(NSError * _Nullable))completion timeout:(long long)timeout desiredEndpointFilter:(MCEndpointFilter * _Nullable)desiredEndpointFilter;
 
 /**
- * @brief (async) Verifies that a connection exists with this CastingPlayer, or triggers a new session request. If the
- * CastingApp does not have the nodeId and fabricIndex of this CastingPlayer cached on disk, this will execute the user
+ * @brief (async) Verifies that a connection exists with this MCCastingPlayer, or triggers a new session request. If the
+ * MCCastingApp does not have the nodeId and fabricIndex of this MCCastingPlayer cached on disk, this will execute the user
  * directed commissioning process.
  *
  * @param completion - called back when the connection process completes. Parameter is nil if it completed successfully
- * @param desiredEndpointFilter - Attributes (such as VendorId) describing an Endpoint that the client wants to interact
+ * @param desiredEndpointFilter - Attributes (such as VendorId) describing an MCEndpoint that the client wants to interact
  * with after commissioning. If this value is passed in, the VerifyOrEstablishConnection will force User Directed
- * Commissioning, in case the desired Endpoint is not found in the on-device cached information about the CastingPlayer
+ * Commissioning, in case the desired Endpoint is not found in the on-device cached information about the MCCastingPlayer
  * (if any)
  */
 - (void)verifyOrEstablishConnectionWithCompletionBlock:(void (^_Nonnull)(NSError * _Nullable))completion desiredEndpointFilter:(MCEndpointFilter * _Nullable)desiredEndpointFilter;
 
 /**
- * @brief Sets the internal connection state of this CastingPlayer to "disconnected"
+ * @brief Sets the internal connection state of this MCCastingPlayer to "disconnected"
  */
 - (void)disconnect;
 
 - (NSString * _Nonnull)identifier;
 - (NSString * _Nonnull)deviceName;
-- (uint16_t)vendorId;
-- (uint16_t)productId;
-- (uint32_t)deviceType;
+- (NSNumber * _Nonnull)vendorId;
+- (NSNumber * _Nonnull)productId;
+- (NSNumber * _Nonnull)deviceType;
```

Comment:
how come we need to change these? Do we get any inherent benefit by ensuring they are only 16/32 bits?

Reply:
Switched them back to primitive types for clarity on what the underlying type is. I had changed to NSNumber to align with some other darwin examples in the Matter SDK like the [https://github.com/project-chip/connectedhomeip/blob/master/src/darwin/Framework/CHIP/MTRCommissionableBrowserResult.h](https://github.com/project-chip/connectedhomeip/blob/master/src/darwin/Framework/CHIP/MTRCommissionableBrowserResult.h) but there does not seem to be much benefit in the MCCastingPlayer use cases - so using the primitives types now.